modules/gitops-architecture-argocd-agent-namespaces.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * argocd-agent-architecture-overview/gitops-architecture-argocd-agent-namespaces.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-architecture-argocd-agent-namespaces_{context}"]
+= Application Management Across Namespaces
+
+The Argo CD Agent architecture uses the *Applications in any namespace* feature to organize Applications per workload cluster on the control plane.
```

Comment:
Without the diagram and original example, this entire page has been oversimplified to death. Let's just remove it.

Reply:
Fixed.